service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/AdditionalRouter.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.javadsl.server
+
+import java.util
+
+import play.api.inject.Injector
+import play.api.routing.Router
```

Comment:
here we are explicitly choosing the Scala API over the Java one. Although it's intended to be used by Java users. 

The rationale behind it is that Java users will get a generate Scala router anyway. That's what the Play Java generates from a `routes` file and what `akka-grpc` Play integration generates as well. 

At the end of the day, the user has a class that needs to be passed for wiring. Doesn't matter much if it comes from a Scala or Java source file.